src/shared/components/query/QueryContainer.tsx

```diff
@@ -69,7 +69,7 @@ export default class QueryContainer extends React.Component<QueryContainerProps,
         return (
 			<FlexCol padded overflow className={styles.QueryContainer}>
                 {
-                    <UnknownStudiesWarning ids={this.store.unknownStudyIds} />
+                    <UnknownStudiesWarning ids={this.store.unknownStudyIds.result} />

```

Comment:
this is a problem i already fixed in the SPA (waiting for unknonwStudyIds) but any time you check result you should really be first checking status.   We should check status wherever we deference the mobxpromise (i.e. refer to result property)

Reply:
Just to be clear, you can keep what you've done, just do check for status